### PR TITLE
[Scopes] fix class declarations

### PR DESCRIPTION
**Pull request by jasonLaster**
#6400

### Summary of Changes

Classes have two bindings, one in the inner class, one in the outer scope. This caused MappedOriginalRanges to accidentally assume the class binding range was "multiple".

```js
export default function root() {
  let one = 1;

  class Thing {
    one() {
      // fails because `Thing` is not referenced locally
      console.log("pause here");
    }
  }

  class Another {
    method() {
      // works because `Another` is referenced locally
      console.log("pause here", Another, Thing, root);
    }
  }

  new Another().method();
  new Thing().one();
}
```

### Test plan

add a mochitest

### CODE REVIEW

**Comment by jasonLaster** [comment]
I'll go in and update the test references in a bit, but here is a small test case.

**Comment by jasonLaster** [comment]
this should be ready for review

File: src/utils/pause/mapScopes/index.js [hunk 1]
```diff
@@ -279,7 +279,7 @@ function generateClientScope(
 
       return {
         // Flow doesn't like casting 'parent'.
```
Comment (loganfsmyth) [inline src/utils/pause/mapScopes/index.js]: Looks like this comment could be removed then.

File: src/test/mochitest/helpers.js [hunk 2]
```diff
@@ -814,6 +814,8 @@ async function expandAllScopes(dbg) {
 async function assertScopes(dbg, items) {
   await expandAllScopes(dbg);
 
+  // dump(JSON.stringify(items.entries()));
```
Comment (loganfsmyth) [inline src/test/mochitest/helpers.js]: Leftovers

Comment (jasonLaster) [reply src/test/mochitest/helpers.js]: nice catch

**Pull request merged**